src/components/molecules/Monaco/CodeIntel/helmChartFile.ts

```diff
@@ -79,6 +79,9 @@ export const helmFileCodeIntel: CodeIntelApply = {
       if (canFindKeyInValuesFile) {
         const commandMarkdownLinkList: monaco.IMarkdownString[] = [];
         keyPathsInFile.forEach(keyPathInFile => {
+          const value =
+            typeof keyPathInFile.value === 'object' ? JSON.stringify(keyPathInFile.value) : keyPathInFile.value;
```

Comment:
maybe it should be `JSON.stringify(keyPathInFile.value, null, 2);` so the output is "pretty-printed"